Solutions/Menes.Specs/Steps/OpenApiDefaultParameterParsingSteps.cs

```diff
@@ -115,9 +115,15 @@ public async System.Threading.Tasks.Task WhenITryToParseTheDefaultValueAsync()
 
             var context = new DefaultHttpContext();
 
+            this.parameters = await builder.BuildParametersAsync(context.Request, operationPathTemplate!);
```

Comment:
Best practice says there should be a `.ConfigureAwait(false)` at the end of this. Roslynator can fix this here and anywhere else it's missing (line 126) in one go.

Reply:
Fixed